ui/src/dashboards/actions/views.ts

```diff
@@ -117,6 +111,7 @@ export const getViewForTimeMachine = (
     }
 
     dispatch(setActiveTimeMachine(timeMachineID, {view}))
+    dispatch(executeQueries(dashboardID))
```

Comment:
i intended this to mean "whenever a timemachine loads a view, run the queries to populate its data"